website/pages/how-to-guides/aws-resource-limits-and-service-quotas.mdx

```diff
@@ -0,0 +1,140 @@
+---
+title: How to Manage AWS Resource and Usage Limits with CloudQuery and Service Quotas
+tag: tutorial
+description: >-
+  A Guide on how to use open source CloudQuery for cloud utilization and service limits for better operational uptime and troubleshooting.  This guide covers managing AWS Service Quotas and how to monitor limits across cloud resources.
+author: jsonkao
+---
+
+import { HowToGuideHeader } from "../../components/HowToGuideHeader"
+
+<HowToGuideHeader/>
+
+## Introduction
+
+AWS Accounts come with over 200 services and each service has different types of resources.  Benefits of cloud computing include the ease of use, elasticity, and advantages of using these cloud services.  However, cloud services come with usage and resource limits.  Proper management of these usage and resource limits can prevent against availability issues when a resource cannot be created, a service is throttled, and helps with proactive management and troubleshooting of limit constraints that may impact active production applications and development environments. 
```

Comment:
Nit: I would remove the first sentence (`AWS Accounts come with`) and just do `Cloud Services come with usage resource limits, ....` given that who ever is reading that is probably familiar with AWS anyway so will just cut to the chase right away.